tox.ini

```diff
@@ -35,10 +55,14 @@ usedevelop = True
 
 [testenv:metadata-validation]
 deps =
```

Comment:
```suggestion
depends = build-dists
deps =
```

Reply:
@webknjaz Sorry but you are asking me to introduce a feature that added very recently to tox (3.7.0) and current minimal version is 3.5.3

Bumping minimal tox version would be outside the scope of this change.

Reply:
agreed